collect_app/src/test/java/org/odk/collect/android/activities/FormMapViewModelTest.java

```diff
@@ -31,87 +31,87 @@
     }
 
     @Test public void getFormTitle_returnsFormTitle() {
```

Comment:
I'm wondering if we can have a data object that encapsulates the form title and id here. I've been feeling like something along those lines would be helpful for decoupling other `ViewModel`s from the `FormController`. I've been strugging to find a simple place to start with this but this feels like it might be it?

Reply:
I struggle a bit with where those kinds of limited façades into domain objects are simplifying vs. obfuscating. In this case, we don't really need the id, it's just being used in a log that perhaps should be cleaned up. The `activity` knows about the `Form` that is being mapped so it might even make more sense to remove those two methods from the view model and have the activity directly get the values it needs from the `Form` object. What do you think?

Reply:
@lognaturel yeah I like that!